src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.cs

```diff
@@ -137,24 +132,42 @@ public void ExecuteReturnOrTypeCommand(EditorCommandArgs args, Action nextHandle
                 return;
             }
 
+            IList<TextChange> textChanges;
```

Comment:
i believe F#/TS go through these paths.  So might be worth manual validation of those guys.